MODULE.bazel

```diff
@@ -70,14 +70,22 @@ pip = use_extension("//python/extensions:pip.bzl", "pip")
         config_settings = [
             "@platforms//cpu:{}".format(cpu),
             "@platforms//os:linux",
+            "//python/config_settings:_is_py_freethreaded_{}".format(
+                "yes" if freethreaded else "no",
+            ),
         ],
         env = {"platform_version": "0"},
+        marker = "python_version ~= \"3.13\"" if freethreaded else "",
```

Comment:
Pretty sure this should be >= 3.13

Reply:
`~= "3.13"` means `>=3.13 and ==3.*`, so this is technically what we want. There probably won't be Python 4 and the freethreaded by then might not be there. That said, maybe `>=` is more readable.